serverless/functions/api-cli/deploy-function-api.mdx

```diff
@@ -29,85 +29,85 @@ Refer to the [Scaleway Developers website](https://www.scaleway.com/en/developer
 
 1. Run the following command in your terminal to create a functions namespace:
 
-  ```json
```

Comment:
I only fixed indentation on the two functions files so no need for a thorough review on the last two!